packages/block-editor/src/components/index.js

```diff
@@ -73,6 +73,9 @@ export { default as __experimentalLinkControlSearchResults } from './link-contro
 export { default as __experimentalLinkControlSearchItem } from './link-control/search-item';
 export { default as LineHeightControl } from './line-height-control';
 export { default as __experimentalListView } from './list-view';
+/**
+ * @see https://github.com/WordPress/gutenberg/blob/HEAD/packages/block-editor/src/components/media-replace-flow/README.md
+ */
```

Comment:
Any particular reason this comment isn’t located in the imported file?

The same applies to another comment.

Reply:
No good reason. Somehow I thought this was the correct place. I have corrected it.